### ISSUE TITLE
POSIX sh to set color for linux terminals (maybe would work on mac)

### ISSUE THREAD

**Issue by pickfire** [NLKNguyen/papercolor#3]
The current shell script uses bash shell which isn't fast and portable, maybe we could use posix sh and support the other terminal at the same time.

**Comment by NLKNguyen** [comment]
I don't mind deleting the current one for gnome-terminal. It was generated by the old base16 palette. It's outdated now. I'm welcoming alternative options.

**Comment by pickfire** [comment]
@NLKNguyen I don't quite like the new generated base16 palette since it is using weird colors from 17-21. The really do broke a lot of things. https://github.com/chriskempson/base16-shell

Now vim and neovim both supports true color now and I see no reason to mess with the 256 colors anymore other than the 16 colors. https://gist.github.com/XVilka/8346728

**Comment by NLKNguyen** [comment]
I also like to only use 16 colors for terminal themes. While fancy terminals are increasing, suck terminals are still dominant. I use Windows Command Prompt as the lowest bar. My goal for now is to support these suckers :D 

True color on vim/nvim depends on whether the terminal supports true color. On 1.0 PaperColor-Theme, true colors can be downgraded to 256 colors automatically if needed, not perfectly of course, but there will be little concern. The PaperColor (light/dark) theme is still based on 256-color because it's sufficient since the beginning.

**Comment by NLKNguyen** [comment]
What are the common terminals that you see people using a lot? I want to target the most popular ones first.

**Comment by pickfire** [comment]
@NLKNguyen Yes, I hate windows command prompt, sucks a lot especially powershell.

For linux terminals, I can't suggest one since there are tons of it. I use [st](st.suckless.org) personally. For now, there only a few terminal that doesn't support true color in linux.
